async_upnp_client/server.py

```diff
@@ -1073,14 +1073,15 @@ async def _parse_action_body(
     try:
         _, action_name = soap_action.split("#")
         data = await request.text()
+        data = data.replace('"s:', '" s:').replace('"u:', '" u:')
         root_el: ET.Element = DET.fromstring(data)
         body_el = root_el.find("s:Body", NAMESPACES)
         assert body_el
         rpc_el = body_el[0]
     except Exception as exc:
         raise HTTPBadRequest(reason="InvalidSoap") from exc
 
-    if action_name not in service.actions:
+    if action_name not in service.actions.keys():
```

Comment:
`.keys()` is the default so its the same as before but just a bit slower when using `.keys()`